plugins/experimental/memcache/tsmemcache.cc

```diff
@@ -514,7 +514,7 @@ MC::binary_get_event(int event, void *data)
     } else {
       return write_binary_error(PROTOCOL_BINARY_RESPONSE_KEY_ENOENT, 0);
     }
-  } else if (event == CACHE_EVENT_OPEN_READ) {
+  } else if (event == CACHE_EVENT_OPEN_READ) { // lgtm[cpp/constant-comparison]
```

Comment:
Assuming `CACHE_EVENT_OPEN_READ != TSMEMCACHE_EVENT_GOT_ITEM`, this code is unreachable.